### DISCUSSION TITLE
How to set priceFormatter for each vertical axis independently?

### DISCUSSION THREAD

**Discussion by ryancwalsh** [tradingview/lightweight-charts#743] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

My problem is that I'd like to have a chart with 2 vertical axes where one is a price and the other is a %, and I'd like them to be formatted as such. 

For example, see https://jsfiddle.net/3yeszpt5/, which uses ``priceFormatter: (price) => `$${price.toFixed(2)}` ``, which is good for the $ axis but not for the % axis.

![image](https://user-images.githubusercontent.com/2086493/118373179-76003380-b583-11eb-8ff3-a209b92b09e0.png)

**Describe the solution you'd like**

Could `priceFormatter` be a property of the `rightPriceScale` and `leftPriceScale` instead of the chart?

Are there any workarounds I could use in the short-term before the library is updated?

**Additional context**

Thank you so much!

**Comment by timocov** [comment, marked as answer]
You can set up a formatter for every series separately, you don't need to use `localization.priceFormatter` in this case https://jsfiddle.net/p9486hoq/.

**Reply by ryancwalsh** [reply]
@timocov Perfect! Thank you SO much!
